inc/class-provider.php

```diff
@@ -62,7 +109,7 @@ protected function request( array $args ): MediaList {
 		] );
 		$response = json_decode( $response );
 
-		if ( json_last_error() ) {
+		if ( json_last_error() !== JSON_ERROR_NONE ) {
```

Comment:
Why do we nee this? `JSON_ERROR_NONE` is `0`, and so it's falsey...? 🤔

Reply:
It's more explicit, just good practice as far as I understand it.